playwright/_impl/_browser_context.py

```diff
@@ -260,10 +262,7 @@ async def unroute(
             )
 
     def expect_event(
-        self,
-        event: str,
-        predicate: Callable = None,
-        timeout: float = None,
+        self, event: str, predicate: Callable = None, timeout: float = None
```

Comment:
Never change the formatter settings when changing code behavior, it adds a lot of stray changes to the code.